.travis-build-without-test.sh

```diff
@@ -10,24 +10,6 @@ if [[ "$SLUGOWNER" == "" ]]; then
   SLUGOWNER=typetools
 fi
 
-
-# jsr308-langtools
-if [ -d ../jsr308-langtools ] ; then
```

Comment:
The Checker Framework depends on this to build JSR308, so we will need to update the Checker Framework Travis builds, be this is merged.

Reply:
Does the Checker Framework need jsr308-langtools?  I don't think it should use jsr308-langtools, either.

Reply:
The jtreg tests rely on the version of javap.jar in jsr308-langtools.  We can just download the last release for those jtreg tests.

Also, in the fat checker.jar, we use the Maven Central version of javac.jar (org.checkerframework:compiler:2.4.0), because we know we have permission to distribute those class files.

Reply:
Also, jsr308-langtools is used when building the jdk8.jar.  (I haven't tried using standard Java 8 to build this, but we probably could.)

Reply:
Your suggested changes sound good.  It will be nice to eliminate all dependency on jsr308-langtools.  Thanks!